guide/src/class.md

```diff
@@ -61,6 +61,7 @@ python object that can be collector `PyGCProtocol` trait has to be implemented.
 * `weakref` - adds support for python weak references
 * `base=xxx.YYY` - use custom base class. It is not possible to call constructor
 of base class at the moment. `xxx.YYY`, `xxx` - module name, `YYY` class name.
+* `basetype` - adds subclass support
```

Comment:
maybe something like "python classes can inherit from this class", otherwise it is not clear